views/account/account_modify.php

```diff
@@ -0,0 +1,616 @@
+<?php
+/**
+ * PEP Capping 2017 Algozzine's Class
+ *
+ * Displays a page to allow account editing.
+ *
+ * This page provides various sections to allow:
+ * 1. A user to modify their own account settings
+ * 2. An admin/superuser to modify other users'
+ * account settings
+ *
+ * Once the form is filled out, if there are any
+ * errors, they will be displayed upon submission.
+ *
+ * @author Michelle Crawley
+ * @copyright 2017 Marist College
+ * @version 0.3.3
+ * @since 0.1
+ */
+
+global $db, $route, $params, $view;
+
+$employeeid = $params[1];
+
+# Checks if the user trying to edit their own account or if they are an admin/superuser
+if (($_SESSION['employeeid'] != $employeeid) && (!(hasRole(Role::Admin)))) {
+    header('Location: /dashboard');
+    die();
+} else {
+    $loggedInUserRole = $_SESSION['role'];
+
+    # Get employee/people information
+    $db->prepare("get-employee-info", "SELECT people.firstname, people.middleinit, people.lastname, employees.email, employees.primaryphone, employees.permissionlevel " .
+        "FROM employees " .
+        "LEFT JOIN people ON employees.employeeid = people.peopleid
```

Comment:
I think this line should end `people.peopleid" . ` to follow what seems standard for multi-line strings